docs/dev/plugins.rst

```diff
@@ -35,13 +35,13 @@ External plugins
 
 External plugins are standard python modules implementing all the requirements of the standard plugins.
 Plugins can be enabled by adding them to :ref:`settings.yml`'s ``plugins`` section.
-Example external plugin can be found `here <https://github.com/asciimoo/searx_external_plugin_example>`_.
+Example external plugin can be found `here <https://github.com/asciimoo/searxng_external_plugin_example>`_.
```

Comment:
The actual repository is https://github.com/asciimoo/searx_external_plugin_example

Reply:
Fixed in local commit